packages/core/src/build/factory.test.ts

```diff
@@ -6,6 +6,21 @@ const llamaFactoryEventAbiItem = parseAbiItem(
   "event LlamaInstanceCreated(address indexed deployer, string indexed name, address llamaCore, address llamaExecutor, address llamaPolicy, uint256 chainId)",
 );
 
+const FactoryEventSimpleParamsAbiItem = parseAbiItem([
```

Comment:
1. In case of function/event with struct, parseAbiItem can be used https://viem.sh/docs/abi/parseAbiItem#parameters . As parseAbiItem was already in use, I preferred to stick to it.
2. Refactored, camelCase all the way. 🫡